Lib/urllib/request.py

```diff
@@ -102,7 +102,7 @@
 from urllib.parse import (
     urlparse, urlsplit, urljoin, unwrap, quote, unquote,
     _splittype, _splithost, _splitport, _splituser, _splitpasswd,
-    _splitattr, _splitquery, _splitvalue, _splittag, _to_bytes,
+    _splitattr, _splitvalue, _splittag,
```

Comment:
Would it be possible to move `import warnings` at line 942, the last line where it's used? You may also sort the list of imports (lines 84-98).